packages/blocks/src/store/private-selectors.js

```diff
@@ -219,32 +219,27 @@ export function getBlockBindingsSource( state, sourceName ) {
  * @param {string} blockTypeName Block type name.
  * @return {boolean} Whether block type has content role attribute.
  */
-export const hasContentRoleAttribute = createSelector(
-	( state, blockTypeName ) => {
-		const blockType = getBlockType( state, blockTypeName );
-		if ( ! blockType ) {
+export const hasContentRoleAttribute = ( state, blockTypeName ) => {
+	const blockType = getBlockType( state, blockTypeName );
+	if ( ! blockType ) {
+		return false;
+	}
+
+	return Object.entries( blockType.attributes ).some(
+		( [ , { role, __experimentalRole } ] ) => {
```

Comment:
We can just use `Object.values` if we are not interested in the keys.

Reply:
Updated in [9a1273f](https://github.com/WordPress/gutenberg/pull/65617/commits/9a1273f0db85da8bbefbb02e90d8a31c48ecf9a9).